runtime/calamari/src/lib.rs

```diff
@@ -116,7 +116,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("calamari"),
 	impl_name: create_runtime_str!("calamari"),
 	authoring_version: 1,
-	spec_version: 3141,
+	spec_version: 3142,
```

Comment:
I guess you can bump this to trigger the CI runtime upgrade test, but let's revert it now.
We only bump these on release PRs.

Reply:
Oh and there's a check in the CI that Calamari and Manta runtime versions are kept synchronized.

Reply:
Just saw it failed - neat :)

Reply:
reverted